Update Game.cs

Thires opened the pull request:
commented out the line #592 sTmp = ParseSubstitutions(sTmp);
stopped the double sub effect in areas <preset>

I am sure that it can be completely removed

jingounchained commented:
This is a lot of change to a lot different things. Can you split these into different PRs and step me through them? And I've considered making the time format a setting (I thought I had done something somewhere with that) but I don't want to just change it outright and potentially break peoples regex that might be matching it right now

Thires commented:
> 



> This is a lot of change to a lot different things. Can you split these into different PRs and step me through them? And I've considered making the time format a setting (I thought I had done something somewhere with that) but I don't want to just change it outright and potentially break peoples regex that might be matching it right now

>The time format, was going to add another option, time stamp and time stamp 24. the way I put it also added seconds as well.
Most of my changes really had to deal with formmain designer, I changed the sizes back to smaller genie 3 settings.

Changed the automapper background color from that cream or whatever to a grey/silver, I did play around with have a color picker option not included in this.

Minor fixes to if statements to have that space.

Everything else is pretty much from pushes.

Thires commented:
The game file was to help stop the double sub from happening, this sTmp = ParseSubstitutions(sTmp); is not needed and it causes the issue.

The time format, was going to add another option, time stamp and time stamp 24. the way I put it also added seconds as well.
Most of my changes really had to deal with formmain designer, I changed the sizes back to smaller genie 3 settings.

Changed the automapper background color from that cream or whatever to a grey/silver, I did play around with have a color picker option not included in this.

Minor fixes to if statements to have that space.

Everything else is pretty much from pushes.
